From 51c25918ec7f911a3ea12a963957b1ac18af84ff Mon Sep 17 00:00:00 2001
From: LuK1337 <priv.luk@gmail.com>
Date: Thu, 16 Aug 2018 12:24:43 +0200
Subject: [PATCH 01/37] SystemUI: Disable config_keyguardUserSwitcher on
 sw600dp

* It's not behaving properly and I don't feel like fixing it.

Change-Id: I634a9abe99299d66a94f9d1ff7d8011476646933
---
 packages/SystemUI/res/values-sw600dp/config.xml | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/packages/SystemUI/res/values-sw600dp/config.xml b/packages/SystemUI/res/values-sw600dp/config.xml
index f91af03f588..5d20564a3ab 100644
--- a/packages/SystemUI/res/values-sw600dp/config.xml
+++ b/packages/SystemUI/res/values-sw600dp/config.xml
@@ -26,9 +26,6 @@
     <!-- The number of columns that the top level tiles span in the QuickSettings -->
     <integer name="quick_settings_user_time_settings_tile_span">1</integer>
 
-    <!-- Set to true to enable the user switcher on the keyguard. -->
-    <bool name="config_keyguardUserSwitcher">true</bool>
-
     <!-- Nav bar button default ordering/layout -->
     <string name="config_navBarLayout" translatable="false">left;back,home,recent;right</string>
 
-- 
2.17.1

